Apps/frontend/src/components/StartStopButton.svelte

```diff
@@ -2,25 +2,23 @@
   import { createEventDispatcher } from "svelte";
   import Switch from "svelte-toggle";
   import { LINE_COLORS_RGBA } from "../const.js";
+  import { channelConfig } from "../stores.js";
   export let channel;
 
   const dispatch = createEventDispatcher();
 
-  let hasStarted = true;
   let color = LINE_COLORS_RGBA[channel];
 
   const handleStartStop = async () => {
-    hasStarted = !hasStarted;
-    dispatch("startStop", { buttonValue: hasStarted });
+    dispatch("startStop", { buttonValue: !$channelConfig[channel].enabled });
   };
 </script>
 
-<div class="control-panel--bottom_switch" data-cy="startStopSwitch-{channel}">
+<div class="control-panel--bottom_switch">
   <Switch
     hideLabel
-    small
```

Comment:
`small` was removed for some reason. Shouldn't impact page all too much, but UI should be a little borken.

Reply:
![image](https://user-images.githubusercontent.com/81998094/213009693-4d08c53a-e944-4b8c-b1b8-25bf3fc7d431.png)
 UI seems fine

Reply:
Yeah, it is. Left is without small, right is with small:
![image](https://user-images.githubusercontent.com/54808295/213010012-5ed21297-8a9e-45cc-bf4d-d9784be3a7ca.png)

Reply:
So do we keep it or do we make it small again?

Reply:
Made it small due to rendering bug

Reply:
Up to personal preference. If you like it more, keep it.